frontend/src/components/molecules/Versions.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+	<div>
+		<p class="header-version">
+			v{{ frontendVersion }}&nbsp;v{{ data?.versions.backendVersion }}
```

Comment:
It would be better to make it clear that one is the frontend, one is the backend version. Not sure what the best way is, would v1.2.3 (backend v2.3.4)  be too long or ugly?

Reply:
@parv0888 I understood that the agreement was that the version display will be `version 0.0.1/0.0.2` for `frontend/backend`.

This does not seem reflected here.

Reply:
fixed